cli/cmd/install_helm_test.go

```diff
@@ -43,6 +43,21 @@ tracing:
 		chartControlPlane := chartControlPlane(t, ha, addOnConfig, "111", "222")
 		testRenderHelm(t, chartControlPlane, "install_helm_output_addons.golden")
 	})
+
+	t.Run("HA mode with podLabels and podAnnotations", func(t *testing.T) {
+		ha := true
+		addOnConfig := `
+global:
+  podLabels:
+    foo: bar
+    fiz: buz
+  podAnnotations:
+    bingo: bongo
+    asda: fasda
+`
+		chartControlPlane := chartControlPlane(t, ha, addOnConfig, "333", "444")
```

Comment:
Do you mind changing the naming of `addonConfig` here and the argument name in the `chartControlPlane` function declaration as well, since this doesn't apply specifically to to addon configs, but any kind of `values.yml` config change? Could be named something like `configDelta`...